include/wx/gtk/window.h

```diff
@@ -139,6 +139,9 @@ class WXDLLIMPEXP_CORE wxWindowGTK : public wxWindowBase
     // --------------
 
     virtual WXWidget GetHandle() const override { return m_widget; }
+#ifdef __WINDOWS__
+    WXHWND GetHWND() const;
```

Comment:
I'm not sure if it's a good or bad idea to use the same name as for the existing function in wxMSW. On one hand, this would allow referring to it without checking for `__WXGTK__` but OTOH do we really plan to have much code using either wxMSW or wxGTK and having to manipulate the underlying `HWND` directly?

Perhaps we should call this `GTKGetWin32Handle()` instead? Granted, this is much longer but this shouldn't be used very often, hopefully.

Reply:
> but OTOH do we really plan to have much code using either wxMSW or wxGTK and having to manipulate the underlying `HWND` directly?

I can't imagine so.

> Perhaps we should call this `GTKGetWin32Handle()` instead? Granted, this is much longer but this shouldn't be used very often, hopefully.

Making it a different, distinguishable name makes sense to me.

Mostly out of curiousity, what's the logic stylistically behind prefixing it with `GTK`?
Since there's other wxWindowGTK functions that aren't...